Add a "strict" option to the m* actions to return an error if some action fails (#2007)
kuzzleio/kuzzle: doc/2/api/controllers/document/m-get/index.md

- `collection`: collection name
- `index`: index name

### Optional:

> - `strict`: if set, an error will occur if at least one document could not been retrieved <SinceBadge version="auto-version" />

Reviewer: ```suggestion
- `strict`: if set, an error will occur if any of the documents could not be retrieved <SinceBadge version="auto-version" />
```
